.gitignore

```diff
@@ -38,8 +38,6 @@ jspm_packages
 
 # Only apps should have lockfiles
 yarn.lock
-package-lock.json
-npm-shrinkwrap.json
```

Comment:
a lockfile is actively dispreferred so that every build uses the latest deps. there's no value in freezing the dep graph here.

Reply:
Yes, because the benefits of not having a lockfile here far outweigh any potential downsides.